src/NuGet.Clients/NuGet.VisualStudio.Client/source.extension.vsixmanifest

```diff
@@ -35,4 +35,4 @@
     <Asset Type="Microsoft.VisualStudio.Assembly" d:Source="File" Path="Newtonsoft.Json.dll" AssemblyName="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed" />
     <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="NuGetConsole.Host.PowerShell" Path="|NuGetConsole.Host.PowerShell|" />
   </Assets>
-</PackageManifest>
+</PackageManifest>
```

Comment:
I like files ending with a newline. Plus you haven't changed anything else in this file.

Reply:
Fixed.